tests/test_common.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file test_common.h
+ * @author Irfan
+ * @brief common header file for all tests to facilitate uniform logging format
+ *
+ * @copyright
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef SR_TEST_COMMON_H
+#define SR_TEST_COMMON_H
+
+#include <stdarg.h>
+#include "tests/config.h"
```

Comment:
I suppose there is no reason to have 2 test headers, why don't you put these few lines into `config.h.in`? You are free to rename it but the header and the source should ideally have the same name.

Reply:
I will move the header into config.h.in, and rename tests/test_common.c to tests/config.c.
Is that okay?

Reply:
I was rather going to `common.h.in` and `common.h`, that is what we generally use. It is not what I would expect to find a file named `config.c`.